pandas/io/parsers.py

```diff
@@ -656,7 +658,13 @@ def _get_options_with_defaults(self, engine):
         options = {}
 
         for argname, default in compat.iteritems(_parser_defaults):
-            options[argname] = kwds.get(argname, default)
+            value = kwds.get(argname, default)
+
+            if argname == 'mangle_dupe_cols' and value != default:
```

Comment:
I think you want this hard-coded effectively; say the default changes to ``None``